pom.xml

```diff
@@ -281,7 +295,7 @@
             <dependency>
                 <groupId>org.mockito</groupId>
                 <artifactId>mockito-core</artifactId>
-                <version>1.10.19</version>
+                <version>${mockito.verison}</version>
```

Comment:
Hi, this dependency is useless anymore, can we remove it?

Reply:
Sound good, removed.